src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  handleFilter: (filterType: string) => void;
+  filter: string;
+  todos: Todo[] | null;
+  deleteTodo: (todoId: number) => void;
+  activeTodos: number;
+};
+
+export const Footer: React.FC<Props> = ({
+  handleFilter,
+  filter,
+  todos,
+  deleteTodo,
+  activeTodos,
+}) => {
+  const handleClearCompleted = () => {
+    todos?.forEach(todo => {
+      if (todo.completed) {
+        deleteTodo(todo.id);
+      }
+    });
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos} {activeTodos === 1 ? 'item' : 'items'} left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filter === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => {
+            handleFilter('all');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filter === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => {
+            handleFilter('active');
+          }}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: filter === 'completed',
+          })}
```

Comment:
Use `Object.values(your enum for statuses)` and render these options with `map()` method